bindtomac-bridged-bond-httpks.sh

```diff
@@ -17,7 +17,7 @@
 #
 # Red Hat Author(s): Radek Vykydal <rvykydal@redhat.com>
 
-TESTTYPE="network"
+TESTTYPE="network rhel-8-failure"
```

Comment:
Could you add a FIXME comment that refers to https://github.com/rhinstaller/kickstart-tests/issues/447 , so that it's easy to find the debug information when trying to fix this?

Reply:
Done.